### DISCUSSION TITLE
BT-105 in subtype 19

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#918] [Q&A] [answered]
Why is the BT-105 forbidden in subtype 19 since SDK 1.10?

**Comment by YvesJo** [comment, marked as answer]
This is already the case for the old forms. This complies with the rules established with DG GROW and that you may find [in the online documentation](https://docs.ted.europa.eu/eforms/latest/schema/procedure-lot-part-information.html#procedureTypeSection), also information that should appear in the notice is listed in Annex V of Directive 23 which does not refer to the procedure Type.
I invite you to get in touch with DG GROW for further details.

**Reply by Bellimee** [reply]
If the field BT-105 is forbidden in eForms 14, 19, 32, 35 according to the SDK rules, then why does COMMISSION IMPLEMENTING REGULATION (EU) 2019/1780 of 23 September 2019, in table 2, state that the BT-105 field is 'O' (optional) in these eForms? Shouldn't the correct value be that there is no value for field BT-105 in eForm 14, 19, 32, 35, so like it is not Optional cause that it is forbidden?

So currently, the SDK rule and the Commission Implementing Regulation (EU) 2019/1780 are contradictory, aren't they?

**Reply by YvesJo** [reply]
The Regulation Annex may contain information whose implementation leads to unclear or exceptional situations. Following received input, information is carefully reviewed by DG GROW and may require a fix. A fix is implemented once identified and approved, allowing an early flaws detection, before its finalization with the next amendment.
For Business, hence non SDK related, questions, please get in touch directly with DG GROW.
